core/src/main/java/org/apache/accumulo/core/metadata/schema/AmpleImpl.java

```diff
@@ -34,12 +70,212 @@ public AmpleImpl(AccumuloClient client) {
 
   @Override
   public TabletMetadata readTablet(KeyExtent extent, ColumnType... colsToFetch) {
-    Options builder = TabletsMetadata.builder().forTablet(extent);
+    TabletsMetadata.Options builder = TabletsMetadata.builder().forTablet(extent);
     if (colsToFetch.length > 0)
       builder.fetch(colsToFetch);
 
     try (TabletsMetadata tablets = builder.build(client)) {
       return Iterables.getOnlyElement(tablets);
```

Comment:
A very simple alternative to this change, would be to just have a shared method return `TabletsMetadata` (before calling getOnlyElement).  Is there a reason just returning `TabletsMetadata` for readTablets() wouldn't suffice?

The method on the interface could look like this:
`public TabletsMetadata readTablets(KeyExtent extent, ColumnType... colsToFetch)`

Reply:
Nevermind, it doesn't look like this would work with the way that we use TabletsMetadata.